src/err.js

```diff
@@ -0,0 +1,19 @@
+function errorCatch(text, toCase) {
+  const errors = [];
+
+  if (!text) {
+    errors.push({ message: 'Text to convert is required' });
+  }
+
+  if (!toCase) {
+    errors.push({ message: '"toCase" query param is required' });
+  }
+
+  if (!['SNAKE', 'KEBAB', 'CAMEL', 'PASCAL', 'UPPER'].includes(toCase)) {
```

Comment:
Move it to the variable
```suggestion
  if (!....includes(toCase)) {
```